src/devices/bus/rs232/qvt102.cpp

```diff
@@ -0,0 +1,76 @@
+// license:BSD-3-Clause
+// copyright-holders:68bit
+
+#include "emu.h"
+#include "qvt102.h"
+
+qvt102_terminal_device::qvt102_terminal_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
+	: device_t(mconfig, SERIAL_TERMINAL_QVT102, tag, owner, clock)
+	, device_rs232_port_interface(mconfig, *this)
+	, m_qvt102(*this, "qvt102")
+	, m_flow_control(*this, "FLOW_CONTROL")
+{
+}
+
+void qvt102_terminal_device::device_add_mconfig(machine_config &config)
+{
+	QVT102(config, m_qvt102, 0);
+	m_qvt102->rs232_conn_txd_handler().set(FUNC(qvt102_terminal_device::output_rxd));
+	m_qvt102->rs232_conn_rts_handler().set(FUNC(qvt102_terminal_device::route_term_rts));
+	m_qvt102->rs232_conn_dtr_handler().set(FUNC(qvt102_terminal_device::route_term_dtr));
+}
+
+INPUT_PORTS_START(qvt102_terminal)
+
+	PORT_START("FLOW_CONTROL")
+	PORT_CONFNAME(0xf, 1, "Flow control")
+	PORT_CONFSETTING(0x00, "None")
+	PORT_CONFSETTING(0x01, "Terminal DTR to remote CTS")
```

Comment:
You need to attach a `PORT_CHANGED_MEMBER` to this so you can switch the output state if someone changes this while the emulation is running.